test/configCases/split-chunks/issue-17332/index.js

```diff
@@ -0,0 +1,4 @@
+it('should run', async () => {
```

Comment:
![image](https://github.com/webpack/webpack/assets/49502170/9fb3120b-4195-422e-aff7-d7f69ba1686e)


I'm confused about the error messages of CI. I did create a test here.

Reply:
Thank you for the PR, I will finish it